src/challengePaths/2015/CommunityService.ts

```diff
@@ -129,31 +129,37 @@ class Test {
    * @param beCertain Whether we should check council.php instead of mafia to determine whether the test is complete.
    * @returns The output of the prepare function given, or null if the test is already complete.
    */
-  run<T>(prepare: () => T, beCertain = false): T | null {
+  run(prepare: () => void, beCertain = false, maxTurns = Infinity): boolean {
     const finishedFunction = () =>
```

Comment:
`const finishedFunction = beCertain ? this.verifyIsDone : this.isDone`

Reply:
https://github.com/Loathing-Associates-Scripting-Society/libram/pull/231
Doing this unbinds the `verifyIsDone` and `isDone` methods from their `this`, and the current setup is more elegant than `const finishedFunction = (beCertain ? this.verifyIsDone : this.isDone).bind(this)`